index.html

```diff
@@ -209,6 +212,8 @@ <h2>Glossary</h2>
 
 <div class="letter_anchor" id="e">E</div>
 
+<p><dfn data-lt="extended grapheme cluster|extended grapheme clusters" id="def_extended_grapheme_cluster" class="lint-ignore">Extended grapheme cluster</dfn>. A specific type of <a>grapheme</a> defined by [[Unicode]] that most closely matches the mapping of the underlying logical character sequence to a <a>user-perceived character</a>. Extended grapheme clusters are needed because the relationship is not one-to-one between code points and graphemes and because the relationship can be somewhat complex.</p>
```

Comment:
We already have a definition for grapheme cluster in the glossary which talks about EGCs, and i think that the entry for extended GC should not re-explain what a grapheme cluster is, but should just point to the main entry, where it already explains that it is one of _two_ types of grapheme cluster defined by the Unicode Standard, and the one which should be used. The main entry also points to where the reader can find further information.  I think we can do better in linking than to just 'throw a book at' people using [Unicode]. ;-)